sdk/python/kfp/dsl/importer_node.py

```diff
@@ -40,7 +40,7 @@ def build_importer_spec(
   Returns:
     An importer spec.
   """
-  assert bool(pipeline_param_name) != bool(constant_value), (
+  assert bool(pipeline_param_name == None) != bool(constant_value == None), (
```

Comment:
A more correct testing for Non is `pipeline_param_name is None`.
So
```
assert (pipeline_param_name is None) != (constant_value is None), (
```
Another variant might be:
```
assert not(pipeline_param_name is not None and constant_value is not None)
assert not(pipeline_param_name is None and constant_value is None)

```

Reply:
Done.